cypress/component/ErrorBoundary.cy.tsx

```diff
@@ -0,0 +1,25 @@
+import ErrorBoundary from '../../src/components/ErrorBoundary';
+
+const ErrorThrowingComponent: React.FC = function ErrorThrowingComponent() {
+  throw new Error('Test error');
+};
+
+describe('ErrorBoundary', () => {
+  it('should display fallback UI when an error is thrown', () => {
```

Comment:
```suggestion
  it('displays ErrorBoundary correctly', () => {
```
I think we should make this distinction clear, because you are not actually causing an error in the app that would then trigger the error boundary.

Reply:
Well I kinda am doing that by throwin an error in the `ErrorThrowingComponent`.

Reply:
Oh, wait, I didn't realize that. In that case: very cool! Keep doing that :tada: